aspnetcore/security/authorization/customizingauthorizationmiddlewareresponse/sample/AuthorizationMiddlewareResultHandlerSample/MyAuthorizationMiddlewareResultHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Authorization.Policy;
+using Microsoft.AspNetCore.Http;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+
+public class MyAuthorizationMiddlewareResultHandler : AuthorizationMiddlewareResultHandler
+{
+    public async Task HandleAsync(
```

Comment:
![image](https://user-images.githubusercontent.com/3605364/96053065-85b37000-0e1a-11eb-8850-6211a01b49c3.png)

Reply:
Oops, needs to be marked as `override`

Reply:
@pranavkm 

'MyAuthorizationMiddlewareResultHandler.HandleAsync(RequestDelegate, HttpContext, AuthorizationPolicy, PolicyAuthorizationResult)': cannot **_override_** inherited member 'AuthorizationMiddlewareResultHandler.HandleAsync(RequestDelegate, HttpContext, AuthorizationPolicy, PolicyAuthorizationResult)' because it is not marked virtual, abstract, or override	AuthorizationMiddlewareResultHandlerSample	C:\GH\aspnet\docs\1\AspNetCore.Docs\aspnetcore\security\authorization\customizingauthorizationmiddlewareresponse\sample\AuthorizationMiddlewareResultHandlerSample\MyAuthorizationMiddlewareResultHandler.cs	10	Active

Reply:
Let me try this out and I'll get back 👍

Reply:
@pranavkm  it doesn't like override but I can use new to hide inherited member. Has the interface changed between RC1 and RC2, I'm using RC1